CONTRIBUTING.md

```diff
@@ -1,70 +1,103 @@
-## Contributing
+# Contributing
 
 | Component | Build Status |
 | --------- | ------------ |
 | Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
 | Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-### Prerequisites:
+# Prerequisites:
   Install VS 2017 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
-
-### To build:
-=======
-#### If you have recently cloned the repo or did a pull from upstream or did git clean -xdf you need to the following before you do anything else in order to build your projects
+  <br/><br/>
+# Data Plane Libraries
+## TO BUILD:
+### Single Service from Command Line
+1. Open VS 2017 Command Prompt
+2. Navigate to service directory e.g. *"sdk\eventhub"*
+3. Invoke `msbuild /t:Build -r` or `dotnet build`
+
+### Single Service from Visual Studio
+1. Open any data-plane solution e.g. *"sdk\eventhub\Microsoft.Azure.EventHubs.sln"*
+2. Build solution from Visual Studio
+
+### All Dataplane Services from Command Line
+1. Open VS 2017 Command Propmpt
+2. Navigate to repository root directory
+3. Invoke `msbuild eng/data-plane.proj /t:Build -r` or `dotnet build eng/data-plane.proj`
```

Comment:
This will soon be changing after https://github.com/Azure/azure-sdk-for-net/pull/6365.

Reply:
Hey @chidozieononiwu, here is how you build the service now:

```bash
dotnet build eng\service.proj /p:ServiceDirectory=servicebus
```

To build everything:

```bash
dotnet build eng\service.proj /p:ServiceDirectory=*
```

To test, just replace test with build.

Reply:
@mitchdenny perhaps we should set ServiceDirectory to `*` by default in the service.proj file so it doesn't have to be specified to build everything.